src/store/todos.test.js

```diff
@@ -25,38 +38,14 @@ describe(`todos reducer`, () => {
         id: 0
       }
     ]}
-    expect(reducer(previousState, todoAdded({description: 'Use Redux', completed: false, id: 1})).todos).toEqual([
-      {
-        description: 'Run the tests',
-        completed: true,
-        id: 0
-      },
-      {
-        description: 'Use Redux',
-        completed: false,
-        id: 1
-      }
-    ])
+    expect(reducer(previousState, todoAdded({description: 'Use Redux', completed: false, id: 1})).todos).toEqual(initialState.todos)
   });
 
   it(`should be able to change a todo's completion status`, () => {
-    const previousState = {todos: [
-      {
-        description: 'Run the tests',
-        completed: false,
-        id: 0
-      },
-      {
-        description: 'Use Redux',
-        completed: false,
-        id: 1
-      }
-    ]};
-
-    expect(reducer(previousState, todoStatusChanged({id: 0, completed: true})).todos).toEqual([
+    expect(reducer(initialState, todoStatusChanged({id: 0, completed: false})).todos).toEqual([
```

Comment:
Codacy has a fix for the issue: [Replace `reducer(initialState,·todoStatusChanged({id:·0,·completed:·false})).todos` with `⏎······reducer(initialState,·todoStatusChanged({·id:·0,·completed:·false·}))⏎········.todos⏎····`](https://app.codacy.com/gh/JThurston4/tdd-todo/pullRequest?prid=9315756)
```suggestion
    expect(
```